src/main/java/com/google/impactdashboard/database_manager/data_read/DataReadManager.java

```diff
@@ -0,0 +1,36 @@
+package com.google.impactdashboard.database_manager.data_read;
+
+import com.google.impactdashboard.data.project.ProjectIdentification;
+import com.google.impactdashboard.data.recommendation.Recommendation;
+import java.util.List;
+import java.util.Map;
+
+/** Interface for retrieving data from the database. */
+public interface DataReadManager {
+  /** 
+   *  Returns a list containing identifying information for all projects in the 
+   *  IAM Bindings table.  
+   */
+  public List<ProjectIdentification> listProjects();
```

Comment:
This code appears to be synchronous, but the design doc mentions that this call can be made asynchronously, is the asynchronous aspect happening in a future edition?

Reply:
I'll sync with Ionis and implement this in future versions